python-lib/gluonts_forecasts/mxnet_utils.py

```diff
@@ -0,0 +1,47 @@
+try:
+    import mxnet as mx
+except OSError as cuda_error:  # error when importing mxnet
```

Comment:
Could we target a more specific error instead? I am thinking of: https://github.com/apache/incubator-mxnet/blob/1c3fcad78caa7e58f81045d6128435a59222dfdd/python/mxnet/base.py#L86

Reply:
we can't catch a MXNet error here as it would require to first import mx.base.MXNetError and we want to catch the failing import of mxnet.
Here is an example of the error when importing mxnet (from the mxnet-cuXXx package) and no Cuda is installed:
`OSError: libcudart.so.10.0: cannot open shared object file: No such file or directory`

Reply:
OK, just for the sake of good naming, we should not it mxnet_or_cuda_error